CMakeLists.txt

```diff
@@ -119,7 +121,7 @@ set(HEADERS_COMMON
 set(TEST_SOURCES test_basic.c test_detail.c test_skipdata.c test_iter.c)
 
 ## architecture support
-if (CAPSTONE_ARM_SUPPORT)
+if (NOT CAPSTONE_X86_ONLY AND CAPSTONE_ARM_SUPPORT)
```

Comment:
all this seems unrelated to the pr

Reply:
he ported his pull req from older branch, so mixed up with newer code. this is from old unrelated commit, but still shows up here